From 11e842a72cc0818d8fe1df9bd0b975b96aace34b Mon Sep 17 00:00:00 2001
From: Sam Thursfield <sam@afuera.me.uk>
Date: Wed, 27 May 2020 13:07:58 +0200
Subject: [PATCH 1/3] Fix build failure due to undefined M_PI constant

The <math.h> header needs to be included.

Previously I suppose libtracker-sparql.h pulled this in.
---
 src/photos-utils.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/photos-utils.c b/src/photos-utils.c
index 1d7be577..d659b47d 100644
--- a/src/photos-utils.c
+++ b/src/photos-utils.c
@@ -32,6 +32,7 @@
 #include <glib.h>
 #include <tracker-sparql.h>
 #include <libgd/gd.h>
+#include <math.h>
 
 #include "photos-application.h"
 #include "photos-device-item.h"
-- 
2.28.0

